mkosi/sandbox.py

```diff
@@ -101,7 +101,7 @@ class cap_user_data_t(ctypes.Structure):
 
 
 def terminal_is_dumb() -> bool:
-    if not sys.stdout.isatty() and not sys.stderr.isatty():
+    if not sys.stdout.isatty() or not sys.stderr.isatty():
         return True
 
     return os.getenv("TERM", "") == "dumb"
```

Comment:
Let's maybe shorten this down to this
```suggestion
    return not sys.stdout.isatty() or not sys.stderr.isatty() or os.getenv("TERM", "") == "dumb"
```
since setting `TERM=dumb` should be enough even if attached to a TTY, that would be consistent with e.g. `tput`.